docs: add Apify AI launch coverage (#2633)
apify/apify-docs: sources/platform/account/apify-ai.md

---
title: Apify AI
description: Apify AI is the conversational AI interface inside Apify Console for finding and running Actors using natural language, instead of keyword search.
sidebar_position: 2
category: platform
slug: /account/apify-ai
---

> Apify AI is a conversational AI interface inside [Apify Console](https://console.apify.com) that lets you describe what you want in natural language and runs the right [Actor](/actors) for you. It uses the same search and execution backend as [Apify Store](https://apify.com/store) search and the [Apify MCP server](/integrations/mcp).

Reviewer: `describe what you want` sounds a lil bit rough 😅 How about stating in the beginning that you can use natural language to find and run Actors.

:::note Apify AI is in beta

Apify AI is currently focused on finding and running Actors. Its capabilities and entry points will expand as it evolves.

:::

> ## Where to find Apify AI

Reviewer: ```suggestion
## Access Apify AI
```
We should keep the headings shorter and more action-oriented

Apify AI has two entry points in Apify Console:

- _Apify Store search bar_ - long, intent-heavy queries go to Apify AI automatically, and short keyword queries stay on the regular Apify Store search. The suggestions below the bar always let you choose either option yourself.
- _Dashboard widget_ - a chat widget on the Console dashboard.

> ![Apify Store search bar with the placeholder Search Actors or ask Apify AI](./images/apify-ai-store-search.webp)

Reviewer: This screentshot doesn't make much sense. Let's either get rid of it or rework it to follow the guidelines (full context, explainers, etc.)

Author: I've added a bit of context, also added a second screen for the Dashboard.

## What you can do with Apify AI

> Apify AI currently supports the core discovery and execution workflow:

Reviewer: Can't we just say `supports the following actions`?

- Search Apify Store for an Actor that matches your goal.
- View an Actor's details, including its [input schema](/actors/development/actor-definition/input-schema) and [pricing](/actors/running/actors-in-store).
- Run an Actor with inputs that Apify AI fills in from your description.
- Fetch results from the Actor's default [dataset](/storage/dataset).
- Summarize the results or answer questions about them, right in the chat.

> Apify AI always asks for your confirmation before it runs an Actor - no run starts, and nothing is charged, without your approval. You can also rate any response with a thumbs up or down to help improve Apify AI.

Reviewer: ```suggestion
Apify AI always asks for your confirmation before it runs an Actor. Nothing is charged without your approval. To help improve Apify AI, you can also rate its responses with a thumbs up or down.
```

## Excluded Actors

Apify AI can run most Actors in Apify Store, but excludes two categories:

> - _Full-permission Actors_ - excluded for security. Running a [full-permission Actor](/actors/running/permissions#full-permission-actors) is a decision you approve personally, so an LLM can't make it on your behalf.

Reviewer: In the previous section it's stated that every run must be approved anyway, so what's the difference here?

Author: I've tried to explain it better now. For security reasons, we don't want Actors with full access to your account to be controlled by an LLM at all.

- _Rental Actors_ - excluded because their subscription-based model doesn't fit the sporadic, on-demand way Apify AI runs Actors.

## Out of current scope

Apify AI focuses on finding and running Actors, so some capabilities aren't in the chat yet. Use Apify Console directly for operations such as:

- Creating or running [tasks](/actors/running/tasks)
- Setting up [schedules](/actors/running/schedules)
- [Publishing](/actors/publishing) Actors
- Configuring external [integrations](/integrations)

> Apify AI also has no chat history or long-term memory yet, and it can't access account data such as your billing usage.

Reviewer: ```suggestion
Apify AI also has no chat history or long-term memory, and it can't access account data such as your billing usage.
```
Too many promises 😅

## Daily usage limit

Apify AI has a daily token limit per user:

- [Free plan](https://apify.com/pricing) users get a lower daily allowance.

> - Users on any paid plan share the same higher daily allowance.

Reviewer: ```suggestion
- Users on paid plans share the same higher daily allowance.
```
